languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -10069,6 +10069,35 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
+      <rulegroup id="ACCORD_TOUT_ADJECTIF" name="accord tout adjectif" default="temp_off">
+          <rule>
+              <pattern>
+                  <token postag="(P\+)?D.*" postag_regexp="yes"/>
+                  <token postag="N f s" postag_regexp="yes"/>
+                  <marker>
+                      <token>tout</token>
+                  </marker>
+                  <token postag="J f s" postag_regexp="yes"><exception regexp="yes">^[aeiou].*|&H_ASPIRE;</exception></token>
+              </pattern>
+              <message>Vérifiez l’accord entre « \1 » et « \3 ».</message>
+              <suggestion>toute</suggestion>
+              <example correction="toute">Une veste <marker>tout</marker> neuve.</example>
+          </rule>
+          <rule>
```

Comment:
This rule catches the following sentence in the rule editor:

Quelqu'un a t-il des infos **toute** récentes sur le glacier de Saint-Sorlin?

Are you sure the correct suggestion is "toutes" and not "tout"?

Reply:
Could you please give me feedback (in the form of a comment) on this question?

Reply:
Yes, the correction is correct so no AP needed.